test/k6/src/api/notifications/orders.js

```diff
@@ -0,0 +1,32 @@
+import http from "k6/http";
+
+import * as config from "../../config.js";
+
+import * as apiHelpers from "../../apiHelpers.js";
+
+export function getById(id, token) {
+    var endpoint = config.notifications.orders_fromId(id);
+    return getByUrl(endpoint, queryParams, token);
```

Comment:
queryParams is not defined here, is it a global? Do we want globals?

Reply:
Nope, gammel moro fra annet api. fikser.